onnx/backend/test/case/node/trilu.py

```diff
@@ -0,0 +1,455 @@
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np  # type: ignore
+
+import onnx
+from ..base import Base
+from . import expect
+
+
+def triu_reference_implementation(x, k=0):
+    return np.triu(x, k)
+
+
+def tril_reference_implementation(x, k=0):
+    return np.tril(x, k)
+
+
+class Trilu(Base):
+    @staticmethod
+    def export_triu():  # type: () -> None
+        node = onnx.helper.make_node(
+            'Trilu',
+            inputs=['x'],
+            outputs=['y'],
+        )
+
+        x = np.random.randint(10, size=(4, 5))
```

Comment:
```suggestion
        x = np.random.randint(10, size=(4, 5)).astype(np.int64)
```
To prevent issues like this: https://github.com/onnx/onnx/pull/3271, please specify numpy integer type for all `np.random.randint`. Thank you!